package.json

```diff
@@ -60,7 +60,7 @@
     "appImage": "yarn sedtoAppImage; yarn build-release-non-linux; yarn sedtoDeb",
     "sedtoAppImage": "sed -i 's/\"target\": \\[\"deb\", \"rpm\", \"freebsd\"\\]/\"target\": \"AppImage\"/g' package.json",
     "sedtoDeb": "sed -i 's/\"target\": \"AppImage\"/\"target\": \\[\"deb\", \"rpm\", \"freebsd\"\\]/g' package.json",
-    "ready": "yarn build-everything && yarn lint-full && yarn test",
+    "ready": "yarn yarn-deduplicate yarn.lock --fail && yarn build-everything && yarn lint-full && yarn test",
```

Comment:
We should add a command `"dedupLock": yarn-deduplicate yarn.lock` I think (name it whatever you want).

Then we can have

```
"lint-full": "yarn dedupLock && yarn format-full && eslint .",
"ready": "yarn dedupLock --fail && yarn build-everything && yarn lint-full && yarn test",
```